core/designsystem/src/main/java/com/dev/innerview/core/designsystem/component/FloatingActionButton.kt

```diff
@@ -0,0 +1,52 @@
+package com.dev.innerview.core.designsystem.component
+
+import android.content.res.Configuration
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material3.ExtendedFloatingActionButton
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.tooling.preview.Preview
+import com.dev.innerview.core.designsystem.theme.InnerViewTheme
+
+@Composable
+fun InnerViewFloatingActionButton(
```

Comment:
🚫 **[ktlint]** <standard:function-naming> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function name should start with a lowercase letter (except factory methods) and use camel case
<!-- __reviewdog__:ChBhNjVjMDgxZTZkNWQ4NDI5EgZrdGxpbnQ= -->